dist/index.html

```diff
@@ -4,8 +4,9 @@
         <meta charset='utf-8'>
         <title>Hootenanny</title>
         <link rel="icon" href="favicon.ico"/>
-        <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+        <link rel="stylesheet" href="../css/80_app.css">
```

Comment:
I think because all css referenced in the source code gets bundled into `iD.css`, this can be plucked. I just commented this out on my comp and seems to work fine.